.github/workflows/ci.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         with:
           swap-size-gb: 7
       - name: Check Website Generation
-        run: ./sbt docs/docusaurusCreateSite
+        run: ./sbt docs/compileDocs
```

Comment:
btw does this also test _building_ the actual website?

Reply:
No, not the whole! Right now it only checks the compilation of documents using mdoc. The next level is to check the validation of sidebar.js that is not included in this version of zio-sbt-website plugin.